### PR TITLE
memepage + my essay addition

### PR DESCRIPTION
**Pull request by naluconcepcion**

### CODE REVIEW

File: essays.html [hunk 1]
```diff
@@ -3,13 +3,10 @@
...
     <meta charset="utf-8" >
     <title>CSC630: The Open Source Movement</title>
-<<<<<<< HEAD
     <link rel="stylesheet" type="text/css" href="./assets/styles/main.css" />
-=======
     <link rel="stylesheet" type="text/css" href='./assets/styles/main.css' />
->>>>>>> 7012f5eb49957b8b13f334495cffcb1f531f2ed6
     <meta name="description" content="CSC630: The Open Source Movement, Phillips Academy Andover, Winter 2018"/>
-    <meta name="keywords" content="computer science, open source" />
+    <meta name="keywords" content="computer science, open sorce" />
```
Comment (milesmcc) [inline essays.html]: typo

File: essays.html [hunk 2]
```diff
@@ -3,13 +3,10 @@
 <head>
     <meta charset="utf-8" >
     <title>CSC630: The Open Source Movement</title>
-<<<<<<< HEAD
     <link rel="stylesheet" type="text/css" href="./assets/styles/main.css" />
-=======
     <link rel="stylesheet" type="text/css" href='./assets/styles/main.css' />
->>>>>>> 7012f5eb49957b8b13f334495cffcb1f531f2ed6
```
Comment (milesmcc) [inline essays.html]: Might want to rebase; this has been fixed iirc and may cause _another_ merge conflict.

File: assignments.html [hunk 3]
```diff
@@ -22,6 +22,7 @@ <h1><a href='index.html'>CSC630: The Open Source Movement</a> // Assignments <sp
             <a href='git.html'>Git Cheatsheet</a>
             <a href='essays.html'>Essays</a>
             <a href="projects.html">Projects</a>
+            <a href="memes.html">Memes</a>
```
Comment (milesmcc) [inline assignments.html:25]: 👍

**Review by milesmcc — COMMENTED** [review]
Great idea; there seems to be one minor typo worth fixing

**Comment by ruddfawcett** [comment]
Thanks, @naluconcepcion!

**Pull request merged**